coalib/bearlib/aspects/Redundancy.py

```diff
@@ -0,0 +1,229 @@
+from coalib.bearlib.aspects import (
+    Aspect, AspectDocumentation, AspectSetting, Root)
+
+
+Root.new_subaspect(
+    'Redundancy',
+    doc=AspectDocumentation(
+            definition="This aspect describes redundancy in your source code.",
+            importance_reason="Redundant code makes your code harder to read"
+                              "and understand.",
+            fix_suggestions="Redundant code can usually be removed without "
+                            "consequences."
+        )
+    )
+
+Root.Redundancy.new_subaspect(
+    'Clone',
+    doc=AspectDocumentation(
+            definition="""
+            Code clones are multiple pieces of source code in your
+            codebase that are very similar.
+            """,
+            importance_reason="""
+            Code clones make editing more difficult due to unnecessary increases
+            in complexity and length.
+             """,
+            fix_suggestions="""
+            Usually code clones can be simplified to only one occurrence. In a
+            lot of cases, both just use different values or variables and can
+            be reduced to one function called with different parameters or
+            loops.
+             """
```

Comment:
this is a leaf, can you fill out *all* docs for this? Also the indentation ofthis particular line is wrong (applies also to others)